packages/web-components/@sfx/base/test/base.test.ts

```diff
@@ -0,0 +1,94 @@
+import { expect, spy, stub } from './utils';
+import Base from '../src/base';
+import { LitElement } from 'lit-element';
+import { TemplateResult } from 'lit-element';
+import * as BaseUtils from '../src/utils';
+
+
+describe('Base Class', () => {
+  let base: any = {}
```

Comment:
No need to type or initialize.

```suggestion
  let base;
```